### ISSUE TITLE
Adaptive Cards- Home- Inputs]: Talkback not announcing role and position of the menu/list items(Red, Green, blue).

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#89]
### Target Platforms

Android

### SDK Version

2023.12.12.1 (85454887)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Android 13
Device: S20
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.12.12.1 (85454887)
Tool: Logitech Bluetooth Keyboard

**Repro Steps:**
1. Turn on Talkback.
2. Open Adaptive Cards Visualizer Application.
3. Home Page will opened.
4. Now Navigate to 'Sample card' button and select it.
5. Now navigate to "choose sample" expands/collapse control and select "Inputs" list item.
6. Now Navigate to Visual tab and select it.
7. Now navigate to Inputs card using swipe gesture and expand "What color do you want?" dropdown button.
8. Now Navigate to menu items and observe the issue.

**Actual Result:**
Talkback is not announcing role and position of the menu/list items((Red, Green, blue). Talkback announce like "ticked, Red in list ".
**Expected Result:**
Talkback should announce the role and the position of the menu/list items((Red, Green, blue). Talkback should announce like "ticked, Red 1 of 3, list/menu item ".

**User Experience:**
Knowing the role for a control helps users know what will happen when that object has focus and is selected, as well as what keyboard shortcuts to use when interacting with the control. When the component’s state/value/property is not defined, assistive technology users are often blocked from using the control or knowing what to expect when interacting with the component.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Talkback is not announcing role and position of the menu list items](https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/a0a0beb0-24f0-4e63-bd76-27b1e27bda68)

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/f9d1299e-0c80-411a-8487-66e760ca35bc

### Card JSON

```JSON
NA
```
